templates/cluster-template-machinepool-system-assigned-identity.yaml

```diff
@@ -162,6 +162,12 @@ metadata:
 spec:
   identity: SystemAssigned
   location: ${AZURE_LOCATION}
+  strategy:
```

Comment:
are these required changes to the templates or are they illustrative / used as examples? I don't see any docs updates or additions in this PR, what parts of the PR do you think are user-facing changes that need documentation updates?

Reply:
These are not required. I only added them for illustrative purposes.

The item that needs documentation is the rollout and delete policies. I'll create an issue to address rather than adding to an already enormous PR.